protocol/0056-REWA-rewards_overview.md

```diff
@@ -98,25 +184,88 @@ Reward accounts and balances must be saved in [LNL checkpoints](./0073-LIMN-limi
+The entire reward account balance is paid out every epoch unless the total value of the metric over all entities is zero, in which case the balance will also be zero anyway (there are no fractional payouts).
 
-```math
-[p_1, m_1]
-[p_2, m_2]
-...
-[p_n, m_n]
-```
+Rewards are first [distributed amongst entities](#distributing-rewards-amongst-entities) (individuals or teams) and then any rewards distributed to teams are [distributed amongst team members](#distributing-rewards-amongst-team-members).
+
+### Distributing rewards amongst entities
+
+Rewards are distributed amongst entities based on the distribution method defined in the recurring transfer.
+
+The protocol currently supports the following distribution strategies:
+
+- [pro-rata](#distributing-pro-rata) by reward metric
+- [exponential-decay](#distributing-with-exponential-decay) by reward metric
+
+#### Distributing pro-rata
+
+Rewards funded using the pro-rata strategy should be distributed pro-rata by each entities reward metric scaled by any active multipliers that party has, i.e.
+
+Let:
+
+- $d_{i}$ be the payout factor for entity $i$
+- $r_{i}$ be the reward metric value for entity $i$
+- $M_{i}$ be the sum of all reward payout multipliers for entity $i$ (reward payout multipliers include the [activity streak multiplier](./0086-ASPR-activity_streak_program.md#applying-the-activity-reward-multiplier) and [bonus rewards multiplier](./0085-RVST-rewards_vesting.md#determining-the-rewards-bonus-multiplier)).
+- $s_{i}$ be the share of the rewards for entity $i$
+
+$$d_{i}=r_{i} M_{i}$$
+
+Note if the entity is a team, $M_{i}$ is set to `1` as reward payout multipliers are considered later when distributing rewards [amongst the team members](#distributing-rewards-amongst-team-members).
+
+Calculate each entities share of the rewards, $s_{i}$ pro-rata based on $d_{i}$, i.e.
+
+$$s_{i} = \frac{d_{i}}{\sum_{i=1}^{n}d_{i}}$$
+
+#### Distributing with exponential decay
+
+Rewards funded using the exponential-decay strategy should be distributed as follows.
+
+1. Calculate each entities reward metric
+2. Order each entity in a descending list by their reward metric value and determine their "rank" in the list
+3. Normalise the rank of each party within a range of 0 to 1, using the highest rank as the upper limit for normalisation.
+4. Calculate each entities share of the rewards using the below formula.
+
+Let:
+
+- $d_{i}$ be the payout factor for entity $i$
+- $s_{i}$ be the share of the rewards for entity $i$
+- $k$ be the decay factor specified in the recurring transfer funding the reward. This must be a float in the range $0\leq k \leq 2$. 
+- $R_{i}$ be the normalised rank of entity $i$
```

Comment:
Did we say how it's normalised somehwere?

Reply:
Step 3 on line 227
> 3. Normalise the rank of each party within a range of 0 to 1, using the highest rank as the upper limit for normalisation.